code/WorkInProgress/Cael_Aislinn/Rust/fuel_injector.dm

```diff
@@ -256,7 +256,7 @@
 	set name = "Rotate Generator (Clockwise)"
 	set src in view(1)
 
-	if (usr.stat || usr.restrained()  || anchored || (usr.status_flags & FAKEDEATH))
+	if (usr.isUnconscious() || usr.restrained()  || anchored)
 		return
 
 	src.dir = turn(src.dir, -90)
```

Comment:
Okay, no, this is not an atomic commit.  

http://ss13.pomf.se/wiki/index.php/Getting_Your_Pull_Accepted#5._Pull_Requests_Must_Be_Atomic

You have unrelated bugfixes and a feature addition crammed into one PR.  We are not congress. You do not add "riders".